docs/src/modules/nixos/core/remote.md

```
@@ -98,3 +100,49 @@ For hyprlang mode, the old string forms (`exec-once`, `monitor = "HEADLESS-2,dis
 - Two sub-features are independent. You can enable streaming without RDP, or RDP without streaming.
 - Sunshine persistence and Hyprland settings are only added when Home Manager is present in system configuration.
 - Hyprland-specific Sunshine application entries are only added when both streaming and Hyprland are enabled.
+
+______________________________________________________________________
```

markdownlint-cli2 (0.22.1)

[warning] Horizontal rule style
Expected: ---; Actual: ______________________________________________________________________

(MD035, hr-style)